sentry-android-fragment/api/sentry-android-fragment.api

```diff
@@ -20,28 +21,19 @@ public final class io/sentry/android/fragment/FragmentLifecycleIntegration : and
 	public fun register (Lio/sentry/IHub;Lio/sentry/SentryOptions;)V
 }
 
-public final class io/sentry/android/fragment/SentryFragmentLifecycleCallbacks : androidx/fragment/app/FragmentManager$FragmentLifecycleCallbacks {
```

Comment:
I understand your reasoning behind this, but we'd not like to introduce breaking API changes outside of major releases. So I'd suggest to revert this to not delay this PR until the next major release.

Also, it is possible that someone has used the `SentryFragmentLifecycleCallback` class directly, to manually call `registerFragmentLifecycleCallbacks` on the fragment manager, so we'd be breaking those.

Reply:
Ok 👍 Do you want to include this change in the next major release? I can prepare PR for it after merging this one

Reply:
hmm, yeah we can do that, though I'm not sure when is the next major, could very well be in a few months 😅